src/recon_buildblock/ProjMatrixByBinUsingRayTracing.cxx

```diff
@@ -676,15 +678,16 @@ calculate_proj_matrix_elems_for_one_bin(
 
       phi = proj_data_info_noarccor.get_phi(bin);
       s_in_mm = proj_data_info_noarccor.get_s(bin);
+      tantheta = proj_data_info_noarccor.get_tantheta(bin);
     }
 
   }
   
   const float cphi = cos(phi);
   const float sphi = sin(phi);
   
-  const float tantheta = proj_data_info_ptr->get_tantheta(bin);
-  const float costheta = 1/sqrt(1+square(tantheta));
+
+  float costheta = 1/sqrt(1+square(tantheta));
```

Comment:
in fact, these lines can now be reinstated, and all the `tantheta` lines above removed, which will lead to a smaller change.

Reply:
Yeah sorry, I thought I made the changes to fix the white space problem so I marked as resolve. Not too sure if it's me or you who should be marking them as resolved, if it's not resolved just mark them as unresolved again.

As for the git conflict, do I do ```git merge master```? My plan is probably add Daniels changes then delete the "X" orientation code after.